docs/integrations/containers-orchestration/opentelemetry/vmware-opentelemetry.md

```diff
@@ -0,0 +1,262 @@
+---
+id: vmware-opentelemetry
+title: VMWare - OpenTelemetry Collector
+sidebar_label: VMWare - OTel Collector
+description: Learn about the Sumo Logic OpenTelemetry app for VMWare.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/containers-orchestration/rabbitmq.png')} alt="icon" width="45"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The VMware app uses unified metrics from the VMware cloud computing virtualization platform to enable monitoring of vCenter, ESXi hosts and individual virtual machines metrics with real-time date displayed in predefined dashboards.
+
+The dashboards provide insight into key metrics such as VM and hosts CPU, memory, disk utilization. This enables you to determine capacity constraints and troubleshoot operational issues related to over-provisioning, changes to configuration, and VM movement.
+
```

Comment:
Can we mention the receiver which we use for collecting data for this app (vcenter in this case) and provide a link to its doc page ?